client/main.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
```

Comment:
This file should be deleted! 
Make sure you fetch changes from master that removes all C++ related files before merging to master

Reply:
Fixed in [aa99708](https://github.com/ghmo2789/puppet-master/pull/59/commits/aa99708d2fbf1aacfaa397298f3f5aec65dab5f3)